### PR TITLE
2.0

### PR DESCRIPTION
**Pull request by dozoisch**
Thinking we can do some clean ups now that hl is a supported render prop.

I also added 12 in the CI.

Anything else you think we should add for 2.0?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,18 +114,18 @@ ReactDOM.render(
...
+__lang__: *DEPRECATED* By default google reCaptcha infers the user's interface language. In order to overwrite the language and update the translation for the reCaptcha widget, you can create a global variable configuring the desired language via `lang`.
+
+*Instead use props.hl* reCaptcha now supports an hl property to be passed at render. The global lang parameter is no longer need and will be removed.
 
 __useRecaptchaNet__: If google.com is blocked, you can set `useRecaptchaNet` to `true` so that the component uses recaptcha.net instead.
 
-__removeOnUnmount__: If you plan to change the lang dynamically, `removeOnUnmount` should probably be `true`. This unloads the google recaptcha script on `componetWillUnmount` to allow for a new google recaptcha script to load next time the reCAPTCHA component is used to facilitate a new language if needed.
+__removeOnUnmount__: *REMOVED* This was only useful for the lang changes. Lang is now changed through
```
Comment (hartzis) [inline README.md]: *REMOVED* This was only useful for the lang changes. Lang is now changed through the `hl` prop.

**Comment by hartzis** [comment]
Thoughts on removing the need for `window.recaptchaOptions` altogether?

File: README.md [hunk 2]
```diff
@@ -114,18 +114,18 @@ ReactDOM.render(
 
 #### Global properties used by reCaptcha
 
-__lang__: By default google reCaptcha infers the user's interface language. In order to overwrite the language and update the translation for the reCaptcha widget, you can create a global variable configuring the desired language via `lang`.
+__lang__: *DEPRECATED* By default google reCaptcha infers the user's interface language. In order to overwrite the language and update the translation for the reCaptcha widget, you can create a global variable configuring the desired language via `lang`.
```
Comment (hartzis) [inline README.md]: If you're thinking about cutting a v2, might as well remove this all together?

File: README.md (hunk 1 above)
Comment (hartzis) [inline README.md]: Same with this, just remove it in v2?

**Comment by hartzis** [comment]
Is it time to include a hook?

**Comment by dozoisch** [comment]
> Thoughts on removing the need for window.recaptchaOptions altogether?

So recaptcha.net support needs to be defined quite early as it currently stands, I wonder if there is a way we can have that without the global options.

**Comment by dozoisch** [comment]
@hartzis published as 2.0! I removed lang completely :). Also added a small upgrade guide.

**Pull request merged**